### PR TITLE
Add maildev to the dev stack

### PR DESCRIPTION
**Pull request by rauchy**
## What type of PR is this? (check all applicable)
- [x] Feature

## Description
[Maildev](https://github.com/djfarrelly/MailDev) adds a simple interface for receiving e-mail in dev mode. This PR adds maildev to the development Docker stack and configures the app to use it. E-mails can be found at `http://localhost:1080`.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
image

### CODE REVIEW

File: redash/settings/__init__.py [hunk 1]
```diff
@@ -210,13 +210,13 @@
...
 # Mail settings:
-MAIL_SERVER = os.environ.get('REDASH_MAIL_SERVER', 'localhost')
+MAIL_SERVER = os.environ.get('REDASH_MAIL_SERVER', 'email')
 MAIL_PORT = int(os.environ.get('REDASH_MAIL_PORT', 25))
 MAIL_USE_TLS = parse_boolean(os.environ.get('REDASH_MAIL_USE_TLS', 'false'))
 MAIL_USE_SSL = parse_boolean(os.environ.get('REDASH_MAIL_USE_SSL', 'false'))
 MAIL_USERNAME = os.environ.get('REDASH_MAIL_USERNAME', None)
 MAIL_PASSWORD = os.environ.get('REDASH_MAIL_PASSWORD', None)
-MAIL_DEFAULT_SENDER = os.environ.get('REDASH_MAIL_DEFAULT_SENDER', None)
+MAIL_DEFAULT_SENDER = os.environ.get('REDASH_MAIL_DEFAULT_SENDER', 'do-not-reply@redash.io')
```
Comment (arikfr) [inline redash/settings/__init__.py]:
```suggestion
MAIL_DEFAULT_SENDER = os.environ.get('REDASH_MAIL_DEFAULT_SENDER', 'john@example.com')
```

**Comment by arikfr** [comment]
:+1:

**Comment by ranbena** [comment]
What's the Redash mail config to enable this?

**Comment by rauchy** [comment]
@ranbena this is enabled by default when running the dev docker-compose stack. Just open http://localhost:1080

**Comment by ranbena** [comment]
@rauchy I mean the Mail config env vars. I assume at least `REDASH_MAIL_DEFAULT_SENDER` is needed.

**Comment by ranbena** [comment]
Oh it's there already 10x!

**Comment by arikfr** [comment]
@ranbena after getting the updated Compose file, you will need to run `docker-compose up -d` (or just `docker-compose up` if that's what you usually do).

**Pull request merged**